charts/generic/pod-delete/experiment.yaml

```diff
@@ -10,22 +10,31 @@ spec:
   definition:
     permissions:
       apiGroups:
```

Comment:
@amitbhatt818 , based on the recent discussions around the need for permissions to be treated as an array - let us make `apigroups/resources/verbs` set as one element.So it will look like: 

```yaml
spec:
  definition:
    permissions:
      - apiGroups:
          - ""
          - "apps"
          - "batch"
          - "litmuschaos.io"
       resources:
          - "deployments"
          - "jobs"
          - "pods"
          - "configmaps"
          - "chaosengines"
          - "chaosexperiments"
          - "chaosresults"
       verbs:
          - "create"
          - "list"
          - "get"
          - "patch"
          - "delete"
      - apiGroups:
          - ""
        resources: 
          - "nodes"
        verbs :
          - "get"
          - "list"
```